.github/workflows/keeper-deploy.yaml

```
@@ -0,0 +1,44 @@
+on:
+  push:
+    branches:
+      - master
+
+name: Deploy Teleport Keeper into Production ECS
+
+jobs:
+  deploy:
+    name: Deploy
+    if: github.ref == 'refs/heads/master'
+    runs-on: ubuntu-latest
+    env:
+      REGION: us-east-1
+      SERVICE_NAME: starknet-teleport-keeper-prod
+
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+      with:
+        submodules: recursive
+
+    - name: Configure AWS credentials
+      uses: aws-actions/configure-aws-credentials@v1
+      with:
+        aws-access-key-id: ${{ secrets.KEEPER_AWS_ACCESS_KEY_ID }}
+        aws-secret-access-key: ${{ secrets.KEEPER_AWS_SECRET_ACCESS_KEY }}
+        aws-region: ${{ env.REGION }}
+
+    - name: Login to container registry
+      id: login-ecr
+      uses: aws-actions/amazon-ecr-login@v1
+
+    - name: Build, tag, and push image to container registry
+      id: build-image
+      env:
+        ECR_REGISTRY: ${{ steps.login-ecr.outputs.registry }}
+        IMAGE_TAG: latest
+      run: |
+        # Build a docker container and
```

```suggestion
        # Build a docker image and
```
I would change this just cause I'm too nit-picky. I also got confused for a sec and had to double check the `docker` commands 😅

---

Makes sense, appreciate the nit-pickiness!